rootfs/etc/systemd/system/dietpi-firstboot.service

```diff
@@ -9,3 +9,6 @@ Type=oneshot
 StandardOutput=tty
 ExecStartPre=
 ExecStart=/bin/dash -c '/DietPi/dietpi/dietpi-firstboot 2>&1 | tee /var/tmp/dietpi/logs/first_boot_install.log'
+
+[Install]
+WantedBy=local-fs.target
```

Comment:
Tiny one: At best use multi-user.target here. The reason is that local-fs.target also runs at some rescue mode and we don't want to have any DietPi scripts running when it is known that login is never reached. Note that it doesn't matter for boot order or something which target you use, as long as the target is planned to be reached by systemd.

I also switched all other DietPi boot services to multi-user.target 1-2 versions ago to be consistent here, so nothing runs aside from base systemd in case something prevents multi-user.target (login prompts).

Reply:
I didn't know about that. Thx

Fixed in a18c393d04ea6e4e1e16024f546faaaa9545c136